src/contentscript/overlay/root/pages/ConnectedAccount/types.ts

```diff
@@ -0,0 +1,40 @@
+export enum Status {
+  Processing = 'Processing',
+  Connected = 'Connected',
+  Error = 'Error',
+}
+
+export interface IUser {
+  img: string
+  name: string
+  origin: string
+  accountActive: boolean
+}
+
+export interface IPair {
+  firstAccount: IUser
+  secondAccount: IUser
+  statusName: Status
+  statusLabel: string // Ok | Time | Attention
+  statusMessage: string
+  closeness: number
+  pendingRequestId?: number
+}
+
+export type TVerificationRequest = {
+  firstAccount: string
+  secondAccount: string
+  isUnlink: boolean
+  firstProofUrl: string
+  secondProofUrl: string
+  transactionSender: string
+}
+
+type TAccountStatus = {
+  isMain: boolean
+}
+
+export type TAccount = {
+  id: string
+  status: TAccountStatus
+}
```

Comment:
I've seen the similar types in `ConnectedAccountService`. I'd reuse it

Reply:
Can we use in the overlay types from the background?

Reply:
Yes, the webpack should deal with it :) It should extract common code and put it into `/build/common.js` module. 
But for project architecture compliance better to move shared types into `src/common/types.ts`.

Reply:
Moved common types to `/src/common/types.ts`